msg/en_US/utils.msg

```diff
@@ -986,6 +986,7 @@ valid options:\n\
   -p, --page=ID               page id; the -L option is required; default : 0\n\
   -v, --verbose               enable verbose status messages; default : disable\n\
   -i, --interval=S            print information every S secs\n
+117 Query result is failed to load. First, wrong directory of log file makes this error. Second, deletion db_ha_apply_info infomation makes this error. \n
```

Comment:
```suggestion
$set 44 MSGCAT_UTIL_SET_APPLYINFO
21 Enter DBA password:
22 The server was not configured for HA.\n
57 HA feature is not supported in this platform.\n
58 applyinfo cannot run as standalone mode.\n
59 There is no query result on db_ha_apply_info due to error caused by invalid log path or deletion of record in db_ha_apply_info. \n
60 \
applyinfo: display CUBRID HA Apply information.\n\
usage: %1$s applyinfo [OPTION] database-name\n\
\n\
valid options:\n\
  -r, --remote-host-name      remote host name; display remote node's active log information \n\
  -L, --copied-log-path=PATH  path of copied log volumes; display copied log information \n\
  -a, --applied-info 	      display applied information; the -L option is required \n\
   -p, --page=ID               page id; the -L option is required; default : 0\n\
   -v, --verbose               enable verbose status messages; default : disable\n\
   -i, --interval=S            print information every S secs\n
```

Most utilities writes messages before usage. and there is no rule for prefixes or numbers.
Please refer to other messages when writing error messages. 

And when writing error messages that are displayed to users through our product, we'd appreciate it if you could just double-check that your sentences are grammatically correct. Simple grammatical errors are easily corrected through google translator.

Also, it would be nice to use words commonly used in this cubrid.msg or util.msg.

Reply:
Right, I searched the usage in the utils.msg and then usage message is almost end of order. Thank you. I will apply it.